includes/class-wc-stripe-helper.php

```diff
@@ -1601,4 +1605,13 @@ public static function get_klarna_preferred_locale( $store_locale, $billing_coun
 
 		return $target_locale;
 	}
+
+	/**
+	 * Checks if the APMs are deprecated. Stripe deprecated them by October 31, 2024 (for the legacy checkout).
+	 *
+	 * @return bool Whether the APMs are deprecated.
+	 */
+	public static function are_apms_deprecated() {
+		return ( new \DateTime() )->format( 'Y-m-d' ) > '2024-10-31' && ! WC_Stripe_Feature_Flags::is_upe_checkout_enabled();
```

Comment:
It should be Oct 29th (the same date mentioned in banner message)
```suggestion
		return ( new \DateTime() )->format( 'Y-m-d' ) > '2024-10-29' && ! WC_Stripe_Feature_Flags::is_upe_checkout_enabled();
```